src/g_editor.c

```diff
@@ -1920,6 +1962,12 @@ static void editor_free(t_editor *x, t_glist *y)
     binbuf_free(x->e_deleted);
     if (x->e_clock)
         clock_free(x->e_clock);
+    t_editor_private *private = x->e_privatedata;
+    freebytes(private, sizeof(*private));
+#ifdef HAVE_KEYBOARDNAV
+        t_kbdnav *kbdnav = &(private->kbdnav);
+        freebytes((void *)kbdnav->kn_linetraverser, sizeof(*kbdnav->kn_linetraverser));
```

Comment:
could be function `kbdnav_free`

Reply:
okay b768c3b1b916e3dce649a30f3ca9209d630fdc18